### DISCUSSION TITLE
Theia IDE 1.50 preview testing

### DISCUSSION THREAD

**Discussion by jfaltermeier** [eclipse-theia/theia#13781] [Release Announcements] [answered]
The new version 1.50.0 of the Theia IDE is available on the preview channel now, please join the preview testing! You can download it here: [Linux](https://download.eclipse.org/theia/ide-preview/1.50.0/linux/TheiaIDE.AppImage), [Mac](https://download.eclipse.org/theia/ide-preview/1.50.0/macos/TheiaIDE.dmg), [Windows](https://download.eclipse.org/theia/ide-preview/1.50.0/windows/TheiaIDESetup.exe). You can update your existing installation by setting the preference `updates.channel` to `preview`. Please respond here when you were able to test the preview without finding blockers, by commenting with a ✔️. If you find any issues, please mention them in this thread and report them as an issue once confirmed by other testers.

**Comment by tsmaeder** [comment]
https://github.com/eclipse-theia/theia/issues/13772 would be reason enough for me to do a patch version. How are you folks experiencing the issue?

**Reply by sgraband** [reply]
I can reproduce the issue. For me it is even longer than 15 seconds

**Reply by xai** [reply]
I agree, running into such a case feels pretty bad as a user.

**Reply by jfaltermeier** [reply]
Since the issue is reproducible in version 1.49.101 of the IDE, addressing it in the upcoming 1.51 release would be acceptable for me as well.

**Comment by sgraband** [comment]
:white_check_mark: I observed no new issues.

**Comment by rschnekenbu** [comment]
:heavy_check_mark:  Looks good to me also

**Comment by tsmaeder** [comment]
I've merged https://github.com/eclipse-theia/theia/pull/13787/. I haven't found any new weirdness with 1.50.0.

**Comment by tsmaeder** [comment]
@jfaltermeier can we get a patch release, please?

**Reply by jfaltermeier** [reply]
You can download the new release here: [Linux](https://download.eclipse.org/theia/ide-preview/1.50.100/linux/TheiaIDE.AppImage), [Mac](https://download.eclipse.org/theia/ide-preview/1.50.100/macos/TheiaIDE.dmg), [Windows](https://download.eclipse.org/theia/ide-preview/1.50.100/windows/TheiaIDESetup.exe) or via the updater

**Comment by rschnekenbu** [comment]
:heavy_check_mark:  1.50.100 update worked fine via the updater. All looks good to me, and opening electron.d.ts is fast as any other file now

**Comment by sgraband** [comment]
✔️ Noticed no new issues and the electron.d.ts opens really fast :+1:

**Comment by xai** [comment]
✔️ Issue is resolved :tada:

**Comment by JonasHelming** [comment]
:white_check_mark: Same for me

**Comment by tsmaeder** [comment]
I haven't found any new problems with the 1.50.100 build. As far as I'm concerned we cant promote it to the "stable" channel. We can always do another patch version if we get relevant fixes before the 27th.

**Reply by jfaltermeier** [reply, marked as answer]
Release is promoted to the stable channel

**Comment by JonasHelming** [comment]
[News & Noteworthy](https://eclipsesource.com/blogs/2024/06/21/eclipse-theia-1-50-release-news-and-noteworthy/)
